### PR TITLE
Update set-up-your-development-environment.md

### LINKED ISSUE #5571: Broken links (404)
The links for windows installers for node.js 10.19.0 are broken.
(https://nodejs.org/dist/latest-v10.x/node-v10.19.0-x64.msi and https://nodejs.org/dist/latest-v10.x/node-v10.19.0-x86.msi)
Working links are https://nodejs.org/dist/v10.19.0/node-v10.19.0-x64.msi and https://nodejs.org/dist/v10.19.0/node-v10.19.0-x86.msi

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 5bee09a8-022e-1b5a-4aab-d6e7c6216808
* Version Independent ID: 49edce1e-f2be-e989-917f-c6b389363e2f
* Content: [Set up your SharePoint Framework development environment](https://docs.microsoft.com/en-us/sharepoint/dev/spfx/set-up-your-development-environment#feedback)
* Content Source: [docs/spfx/set-up-your-development-environment.md](https://github.com/SharePoint/sp-dev-docs/blob/master/docs/spfx/set-up-your-development-environment.md)
* Product: **sharepoint**
* Technology: **sharepoint-framework**
* GitHub Login: @spdevdocs
* Microsoft Alias: **spdevdocs**

### PR DESCRIPTION
**Pull request by JarbasHorst**
#### Category
- [x] Content fix
- [ ] New article

#### Related issues:
- fixes #5571 
- partially #issuenumber
- mentioned in #issuenumber

#### What's in this Pull Request?

Fixed broken nodejs links

### CODE REVIEW

File: docs/spfx/set-up-your-development-environment.md [hunk 1]
```diff
@@ -27,8 +27,8 @@ You can also follow these steps by watching this video on the SharePoint PnP You
 
 Install [NodeJS LTS version 10](https://nodejs.org/dist/latest-v10.x/).
 
-- If you are in Windows, you can use the msi installers ([x86](https://nodejs.org/dist/latest-v10.x/node-v10.19.0-x86.msi) or [x64](https://nodejs.org/dist/latest-v10.x/node-v10.19.0-x64.msi)) in this link for the easiest way to set up NodeJS (notice that these direct links evolve over time, so check the latest v10 from the above directory).
-- If you have NodeJS already installed, check that you have the correct version by using `node -v`. It should return version 10.19.0.
+- If you are in Windows, you can use the msi installers ([x86](https://nodejs.org/dist/latest-v10.x/node-v10.20.1-x86.msi) or [x64](https://nodejs.org/dist/latest-v10.x/node-v10.20.1-x64.msi)) in this link for the easiest way to set up NodeJS (notice that these direct links evolve over time, so check the latest v10 from the above directory).
```
Comment (andrewconnell) [inline docs/spfx/set-up-your-development-environment.md:30]: Can you instead remove the specific links to versions? This is going to be a never-ending update to this doc with every minor & patch version change.

Let's just point to the link where they can get the latest version of v10 & direct them to use the MSI's.

File: docs/spfx/set-up-your-development-environment.md [hunk 2]
```diff
@@ -27,8 +27,8 @@ You can also follow these steps by watching this video on the SharePoint PnP You
 
 Install [NodeJS LTS version 10](https://nodejs.org/dist/latest-v10.x/).
 
-- If you are in Windows, you can use the msi installers ([x86](https://nodejs.org/dist/latest-v10.x/node-v10.19.0-x86.msi) or [x64](https://nodejs.org/dist/latest-v10.x/node-v10.19.0-x64.msi)) in this link for the easiest way to set up NodeJS (notice that these direct links evolve over time, so check the latest v10 from the above directory).
-- If you have NodeJS already installed, check that you have the correct version by using `node -v`. It should return version 10.19.0.
+- If you are in Windows, you can use the msi installers ([x86](https://nodejs.org/dist/latest-v10.x/node-v10.20.1-x86.msi) or [x64](https://nodejs.org/dist/latest-v10.x/node-v10.20.1-x64.msi)) in this link for the easiest way to set up NodeJS (notice that these direct links evolve over time, so check the latest v10 from the above directory).
+- If you have NodeJS already installed, check that you have the correct version by using `node -v`. It should return version 10.20.1.
```
Comment (andrewconnell) [inline docs/spfx/set-up-your-development-environment.md:31]: Same comment as above...

**Comment by PrasadKasireddy** [comment]
Vesa has been updated the Node.JS version 10.20.1 - https://github.com/SharePoint/sp-dev-docs/commit/117e2d3c3bbb418dca55d831767da6a6bc976f2c, 
I assume that we can close this PR

**Comment by JarbasHorst** [comment]
Excuse me for the delay here. I did not notice the comments from Andrew. Please, close this PR and sorry again for the delay.

**Pull request closed without merging**